evm_loader/program/src/evm/stack.rs

```diff
@@ -12,6 +12,7 @@ use crate::{error::Error, types::Address};
 const ELEMENT_SIZE: usize = 32;
 const STACK_SIZE: usize = ELEMENT_SIZE * 128;
 
+#[repr(C)]
 pub struct Stack {
```

Comment:
In this file, `serde::Serialize` and `serde::Deserialize` impls can be removed.

Reply:
done